src/domains/job/recruitment/detail/components/RelatedRecruitmentPosts.jsx

```
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { getAllRecruitmentPosts } from '../../../../job/main/service/jobMainService'
+import JobPostList from '../../../main/components/JobPostList'
+import Spinner from '../../../../../components/web/Spinner'
+
+const RelatedRecruitmentPosts = ({ currentId }) => {
```

ESLint

[error] 'currentId' is missing in props validation

(react/prop-types)